data/navigations_de.yml

```
@@ -0,0 +1,98 @@
+- title: Produkte
+  type: products-group
+  dropdown: products
+  link: /producten.html
+  links:
+    - title: CAPP Learning Management
+      text: Vielfalt des Lernens für motivierte und qualifizierte Mitarbeiter
+      style: item-capp
```

`class` instead of `style`?

```suggestion
      class: item-capp
```

---

Middleman doesn't like 'class' key.

---

`product[:class]`